google/cloud/documentai_toolbox/converters/converter.py

```diff
@@ -15,9 +15,368 @@
 #
 """Document.proto converters."""
 
-from google.cloud.documentai_toolbox.converters.config.converter_helpers import (
-    _convert_documents_with_config,
-)
+from concurrent import futures
+import os
+import time
+from typing import Dict, List, Optional, Set, Tuple
+
+from google.api_core.client_options import ClientOptions
+
+from google.cloud import documentai
+from google.cloud.documentai_toolbox import constants
+from google.cloud.documentai_toolbox.converters.config import bbox_conversion
+from google.cloud.documentai_toolbox.converters.config.block import Block
+from google.cloud.documentai_toolbox.utilities import gcs_utilities
+
+FILES_TO_IGNORE = [".DS_Store"]
+
+
+def _get_base_ocr(
```

Comment:
Are these simply moved from converter_helpers?  Let me know if these additions need a more detailed review.

Reply:
These were refactored as well, and everything was moved into this file.

You can see just the refactoring work in this commit https://github.com/googleapis/python-documentai-toolbox/pull/139/commits/fb517fba8154e8e8da0553b8c8b3467a39061786